databases-analytics/databases/postgresql-tf/README.md

```diff
@@ -0,0 +1,33 @@
+## Create PostgreSQL Database with Terraform
+
+### General information
+ - 🔗 [Using Terraform with OVHcloud](https://help.ovhcloud.com/csm/fr-terraform-at-ovhcloud?id=kb_article_view&sysparm_article=KB0054776)
+ - 🔗 [Find out how to order and manage your Public Cloud managed database service using Terraform ](https://help.ovhcloud.com/csm/fr-public-cloud-databases-order-terraform?id=kb_article_view&sysparm_article=KB0048840)  
+ - 🔗 [How to use Terraform](https://help.ovhcloud.com/csm/en-gb-public-cloud-compute-terraform?id=kb_article_view&sysparm_article=KB0050787)
+ - 🔗 [cloud_project_database](https://registry.terraform.io/providers/ovh/ovh/latest/docs/resources/cloud_project_database)
+ - 🔗 [OVH token generation page](https://www.ovh.com/auth/api/createToken?GET=/*&POST=/*&PUT=/*&DELETE=/*)
+ - 🔗 [public-cloud-databases-examples](https://github.com/ovh/public-cloud-databases-examples)
+
+### Set up
+  - Install the [Terraform CLI](https://www.terraform.io/downloads.html)
+  - Get the credentials from the OVHCloud Public Cloud project:
+    - `application_key`
+    - `application_secret`
+    - `consumer_key`
+  - Get the `service_name` (Public Cloud project ID)
+
+### Demo
+  - set the environment variables `OVH_APPLICATION_KEY`, `OVH_APPLICATION_SECRET`, `OVH_CONSUMER_KEY` and `OVH_CLOUD_PROJECT_SERVICE`
+  - replace the placeholder `<your ip here>` (see https://whatismyipaddress.com/)
```

Comment:
Instead, why not defining an env variable

`export IP="..."`

and then use it in terraform plan/apply and destroy commands? :) 

I tested it and it's working smootly

Reply:
done with https://github.com/ovh/public-cloud-examples/pull/38/commits/f3250c521201b241cea16905de701116af995877